bftengine/src/bcstatetransfer/RangeValidationTree.cpp

```diff
@@ -148,13 +148,13 @@ uint64_t NodeInfo::prevRvbIndex(uint64_t rvb_index, uint8_t level) {
 // This requires us to write our own DigestContext
 const shared_ptr<char[]> RVBNode::computeNodeInitialValue(NodeInfo& node_info, const char* data, size_t data_size) {
   ConcordAssertGT(node_info.id(), 0);
-  DigestUtil::Context c;
+  DigestGenerator digestGenerator;
```

Comment:
In this file the variable names are having underscores. So in this file, the name should be digest_generator;

Reply:
Addressed in https://github.com/arc-vmware/concord-bft/pull/3/commits/b7fc8a50c49f6f986000ec33fefe4d8e1580802d